src/com/lftechnology/java/training/sanish/application/controller/EmployeeController.java

```diff
@@ -0,0 +1,384 @@
+        } finally {
+            DbConnect.dbClose();
+        }
+    }
+
+    /**
+     * Edit employee page
+     *
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void editEmployeePage() {
+        try (Scanner inputScanner = new Scanner(System.in)) {
+            if (UserLogin.isLogin()) {
+                UserDao userDao = new UserDao();
+                UserEmployee userEmployee = new UserEmployee();
+                EditEmployeePage.renderPage(userEmployee, false);
+                int userId = EmployeeHelper.getUserId(inputScanner);
+                if (userId != -1) {
+                    User user = userDao.findById(userId);
+                    Employee employee = userDao.getEmployee(user);
+                    userEmployee.setUser(user);
+                    userEmployee.setEmployee(employee);
+
+                    EmployeeHelper.changeUserName(inputScanner, user);
+                    EmployeeHelper.changeFullName(inputScanner, employee);
+                    EmployeeHelper.changeUserEmail(inputScanner, user);
+                    EmployeeHelper.changeAddress(inputScanner, employee);
+                    EmployeeHelper.changeDepartment(inputScanner, employee);
+                    EmployeeHelper.changeRole(inputScanner, employee);
+
+                    user = userDao.findById(userId);
+                    employee = userDao.getEmployee(user);
+                    userEmployee.setUser(user);
+                    userEmployee.setEmployee(employee);
+                    EditEmployeePage.renderPage(userEmployee, true);
+                }
+
+                int option;
+                option = UserInput.getIntegerNumber(inputScanner, 1, 3);
+                if (option == 1) {
+                    editEmployeePage();
+                } else if (option == 2) {
+                    employeeDashboardPage();
+                } else {
+                    LoginController.logoutPage();
+                }
+            } else {
+                LoginController.LoginPage();
+            }
+        } catch (NumberFormatException e) {
+            LOGGER.log(Level.WARNING, Constants.EXCEPTION_ERROR_MSG_LABEL + "{0}", new Object[] { e });
+        } finally {
+            DbConnect.dbClose();
+        }
+    }
+
+    /**
+     * Terminate user page
+     *
+     * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+     */
+    public static void terminateEmployeePage() {
+        try (Scanner inputScanner = new Scanner(System.in)) {
+            String message = "";
+            if (UserLogin.isLogin()) {
+                UserDao userDao = new UserDao();
+                TerminateEmployeePage.renderPage(false, message);
+                int userId = EmployeeHelper.getUserId(inputScanner);
+                if (userId != -1) {
+                    User user = userDao.findById(userId);
+                    if (EmployeeHelper.terminateConformation(inputScanner, user)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS134)